components/description/Svg.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+
+function Icon() {
+  return (
+    <svg
```

Comment:
Why did you choose to do that with SVG?

Reply:
Because if we want to incorporate it into animation in the future, we'll get more control.
Have any problems with the file?